code/modules/power/smes.dm

```diff
@@ -423,13 +446,27 @@
 	update_icon()
 	..()
 
-/obj/machinery/power/smes/magical

```

Comment:
Please don't remove `/obj/machinery/power/smes/magical`, they're used in several away mission maps.   Also would appreciate if  `outputting = 1` was set on the magical ones.

Reply:
As written on discord, magical SMESes felt kind of deprecated for quite some time (at least i tried to get rid of them back on Bay) as they are one giant hack. I included newest version of customisable power sources for debugging in this PR, which i believe would be a suitable replacement. See /obj/machinery/power/debug_items/infinite_generator - for actual map use that may be encountered by players show_extended_information should be set to FALSE.